docs/pages/kubernetes-access/helm/guides/digitalocean.mdx

```diff
@@ -0,0 +1,174 @@
+First, get external IP for the Kubernetes cluster. The `EXTERNAL-IP` field may take a few minutes to show a public IP address, so continue trying this command until you see a public IP address.
+
+```code
+
+$ kubectl --namespace=teleport-cluster get services
+NAME               TYPE           CLUSTER-IP      EXTERNAL-IP       PORT(S)          AGE
+# teleport-cluster   LoadBalancer   10.245.163.12   144.126.252.144   443:31959/TCP,3023:30525/TCP,3026:30079/TCP,3024:32437/TCP   19m
+
+```
+
+Once you can see the value for `EXTERNAL-IP`(public IP address), update your DNS record such that the clusterName's A record points to this IP address. For example `144.126.252.144` is the public IP in the above case.
+
+## Step 3/4. Create Teleport User
+Now we create a Teleport user by executing the `tctl` command with `kubectl`.
+
+```code
+$ kubectl --namespace teleport-cluster exec deploy/teleport-cluster -- tctl users add tadmin --roles=access,editor --logins=root,ubuntu
+# User "tadmin" has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h:
+# https://tele.teleporters.dev:443/web/invite/9b8c8ccb53d0aab612487d279d3d9831
+
+# NOTE: Make sure tele.teleporters.dev:443 points at a Teleport proxy which users can access.
+```
+
+Copy the link shown after executing the above command and open the link in a web browser to complete user registration process (the link is `https://tele.teleporters.dev:443/web/invite/9b8c8ccb53d0aab612487d279d3d9831` in the above case).
+![Setup User](../../../../img/helm/digitalocean/setup-user.png)
+
+After you complete the registration process by setting up a password and enrolling in two-factor authentication, you will be logged in to Teleport WebUI. 
+
+In this step, we created a user **tadmin** with roles `access, edit`. These are the default roles available in Teleport. However, to allow this user to access the Kubernetes cluster, we will need to assign **tadmin** a role authorized to access the Kubernetes cluster. So first, let's create a role named **member** with the Kubernetes privilege `system:master`.
+
+**Create a new Kubernetes Role name "member":**
+
+Create a file name `member.yaml` with the following spec:
+```code
+$ vi member.yaml
+kind: role
+version: v4
+metadata:
+  name: member
+spec:
+  allow:
+    kubernetes_groups: ["system:masters"]
+
+```
+<Admonition
+  type="warning"
+  title="Warning"
+>
+`system:masters` is a Kubernetes built-in role that is similar to `sudo` permissions in Linux systems. It is used here for demonstration purposes and should be sparingly used in production.
+</Admonition>
+
+Next, create this role in Kubernetes with the command:
+
+```code
+
+$ POD=$(kubectl --namespace=teleport-cluster get pod -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
+$ kubectl --namespace=teleport-cluster exec -i ${POD?} -- tctl create -f < member.yaml
+```
+
+**Assign "member" role to user "tadmin":**
```

Comment:
- I think there should be a "the" before "member"
- I think this should be a heading rather than bold text